application/components/Header/Header.tsx

```diff
@@ -1,16 +1,19 @@
-import Link from "next/link";
+// import Link from "next/link";
+import Image from "next/image";
 import {
-  MagnifyingGlassIcon,
-  MoonIcon,
-  SunIcon
+  MagnifyingGlassIcon
+  // MoonIcon,
+  // SunIcon,
 } from "@heroicons/react/24/solid";
+
 import Icon from "components/Icon/Icon";
 import Input from "components/Input/Input";
 import { useThemeContext } from "context/theme";
 
 export default function Header() {
-  const { darkMode, setDarkMode } = useThemeContext();
-  const DarkModeIcon = !darkMode ? MoonIcon : SunIcon;
+  const { darkMode } = useThemeContext();
+  // const { darkMode, setDarkMode } = useThemeContext();
+  // const DarkModeIcon = !darkMode ? MoonIcon : SunIcon;
```

Comment:
we can keep these commented for now, I'll try and implement feature flags to turn darkMode on/off